docs/source/commandline/commandline-all.rst

```diff
@@ -107,54 +144,76 @@ Note: By default, all the '**-list**' commands below write info to the screen. I
 		* 8: means some files were identified
 		* 9: means no files were identified
 
+.. _mame-commandline-listdevices:
+
 **-listdevices** / **-ld** [<*gamename|wildcard*>]
 
         Displays a list of all devices known to be hooked up to a game.  The ":" is considered the game itself with the devices list being attached to give the user a better understanding of what the emulation is using.
 
+.. _mame-commandline-listslots:
+
```

Comment:
This won't work with a wildcard since it needs to build a machine configuration.

It's also worth noting that as you populate slots, this will show any additional slots they provide.  For example with a computer driver, putting a disk controller card in a slot will expose additional slots for the disk drives.

Reply:
Confirmed to work with wildcards. We discussed this on IRC. I'm going to leave it for the moment-- it may have some weird effects if misused, but it does work and there are use cases for it.